omod/src/main/webapp/fragments/patientdashboard/visits.gsp

```diff
@@ -57,12 +73,39 @@
             encounterCount = ${ param.encounterCount }; // This variable is defined in patientdashboard/patientDashboard.gsp
         <% } %>
         loadTemplates(visitId, ${ patient.id }, fromEncounter, encounterCount);
+
+        <%
+            def visits = patient.allVisitsUsingWrappers;
+            visits.eachWithIndex{ wrapper, idx ->
```

Comment:
Specifically, in the controller we already determine an activeVisit variable: https://github.com/andrewmoko/openmrs-module-coreapps/blob/8b12fcc8558a005e508cc6ea37f70c18be6f0793/omod/src/main/java/org/openmrs/module/coreapps/page/controller/patientdashboard/PatientDashboardPageController.java#L76

Therefore you shouldn't need to loop here in order to find the active visit.